fvm/fvm.go

```diff
@@ -89,6 +89,11 @@ func (vm *VirtualMachine) GetAccount(ctx Context, address flow.Address, v state.
 // captured by the Cadence runtime and eventually disambiguated by the parent context.
 func (vm *VirtualMachine) invokeMetaTransaction(ctx Context, tx *TransactionProcedure, sth *state.StateHolder, programs *programs.Programs) (errors.Error, error) {
 	invocator := NewTransactionInvocator(zerolog.Nop())
+
+	// do not deduct fees or check storage in meta transactions
+	ctx.TransactionFeesEnabled = false
+	ctx.LimitAccountStorage = false
```

Comment:
Looks good! Maybe this should only be done for particular meta transactions, like the transaction fee deduction meta transaction, but not for others?

Reply:
transaction fee deduction is no longer a meta transaction. Only account creation (an bootstrapping which is not relevant here) are still meta transactions.

since meta transactions are sort of like sub-transaction transactions, it also conceptually doesn't make sense for any of them to deduct fees or check storage limits.